io/disk/htx_block_devices.py

```diff
@@ -172,7 +177,9 @@ def test_start(self):
         process.run("htxcmdline -createmdt")
 
         if not os.path.exists(f"/usr/lpp/htx/mdt/{self.mdt_file}"):
-            self.fail(f"MDT file {self.mdt_file} not found")
+            process.run(f"htxcmdline -createmdt -mdt {self.mdt_file}")
+            if not os.path.exists(f"/usr/lpp/htx/mdt/{self.mdt_file}"):
+                self.fail(f"MDT file {self.mdt_file} not found")
```

Comment:
make this fail message as failed to create mdt file, as your checking after create command

Reply:
Good catch @Naresh-ibm . Updated as suggested.